### PR TITLE
Feature/multi sphere

### PR DESCRIPTION
**Pull request by meganrm**
Problem
=======
What is the problem this work solves, including
We want to load format version 1.0 recipes with multi sphere ingredients

Solution
========
A few small fixes to make sure these load

## Type of change
Please delete options that are not relevant.

* New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by mogres** [comment]
Is there a test recipe I could try to pack? Currently running into an error `AttributeError: 'MultiSphereIngr' object has no attribute 'collides_with_compartment'` while trying to pack `cellpack/test-recipes/sphere_tree.json`

File: cellpack/autopack/Compartment.py [hunk 1]
```diff
@@ -1230,7 +1230,9 @@ def BuildGrid_trimesh(self, env, grdPos, vSurfaceArea, srfPts, idarray, mesh_sto
         mesh = mesh_store.get_mesh(self.gname)
         # voxelized
         self.log.info(f"{self.name}: CREATED MESH")
-        trimesh_grid_surface = creation.voxelize(mesh, pitch=env.grid.gridSpacing)
+        trimesh_grid_surface = creation.voxelize(
+            mesh, pitch=env.grid.gridSpacing / 2
+        ).hollow()
```
Comment (meganrm) [inline cellpack/autopack/Compartment.py:1235]: gets a better mesh voxelization especially with low grid spacing

File: cellpack/autopack/interface_objects/representations.py [hunk 2]
```diff
@@ -119,7 +119,7 @@ def get_mesh_path(self):
         else:
             if self.mesh["path"] == "default":
                 return f"{self.DATABASE}/geometries/{self.mesh['name']}"
-            return f"{self.mesh['path']}{self.mesh['name']}"
+            return f"{self.mesh['path']}/{self.mesh['name']}"
```
Comment (meganrm) [inline cellpack/autopack/interface_objects/representations.py:122]: so we don't get concatenated paths

Comment (prismofeverything) [reply cellpack/autopack/interface_objects/representations.py:122]: Could be worth looking into using `pathlib.Path` for paths in general. They handle all the corner cases of concatenation and also help when using windows style paths. 

> from pathlib import Path
> Path('one/path') / 'further/paths'

**Review by prismofeverything — APPROVED** [review]
Looks good to me

**Pull request merged**